### ISSUE TITLE
LICENSE file in package is installed in site-packages/ outside the package subdirectory

### ISSUE THREAD

**Issue by huonw** [youtype/types-s3transfer#2]
The `LICENSE` file is explicitly packaged into the wheel, but is outside the `s3transfer-stubs` directory.  This means it ends up installed directly within `site-packages/` when installed, rather than directly associated with `types-s3transfer`. An unscoped loose file like this can cause confusion and can interfere with other packages that also have a similar mistake (e.g. https://github.com/pantsbuild/pex/issues/1594).

For example, downloading https://files.pythonhosted.org/packages/85/6e/35a35e36fa60ab376ecbc6fadd553a6978443ea8622ecac109f2b17c7c7f/types_s3transfer-0.6.0.post4-py3-none-any.whl (from https://pypi.org/project/types-s3transfer/#files) and running `unzip -l` on it shows the following contents: notice the `LICENSE` listed at the top:

```
LICENSE
s3transfer-stubs/__init__.pyi
s3transfer-stubs/bandwidth.pyi
s3transfer-stubs/compat.pyi
s3transfer-stubs/constants.pyi
s3transfer-stubs/copies.pyi
s3transfer-stubs/crt.pyi
s3transfer-stubs/delete.pyi
s3transfer-stubs/download.pyi
s3transfer-stubs/exceptions.pyi
s3transfer-stubs/futures.pyi
s3transfer-stubs/manager.pyi
s3transfer-stubs/processpool.pyi
s3transfer-stubs/py.typed
s3transfer-stubs/subscribers.pyi
s3transfer-stubs/tasks.pyi
s3transfer-stubs/upload.pyi
s3transfer-stubs/utils.pyi
types_s3transfer-0.6.0.post4.dist-info/LICENSE
types_s3transfer-0.6.0.post4.dist-info/WHEEL
types_s3transfer-0.6.0.post4.dist-info/METADATA
types_s3transfer-0.6.0.post4.dist-info/RECORD
```

Installing that puts the `LICENSE` file within `site-packages/`. e.g.:

```shell
python -m venv /tmp/s3transfer-test
. /tmp/s3transfer-test/bin/activate

# before
ls /tmp/s3transfer-test/lib/python3.9/site-packages/ 
#> _distutils_hack
#> distutils-precedence.pth
#> pip
#> pip-21.2.4.dist-info
#> pkg_resources
#> setuptools
#> setuptools-58.1.0.dist-info

python -m pip install types-s3transfer

# after
ls /tmp/s3transfer-test/lib/python3.9/site-packages/
#> LICENSE 
#> _distutils_hack
#> distutils-precedence.pth
#> pip
#> pip-21.2.4.dist-info
#> pkg_resources
#> s3transfer-stubs
#> setuptools
#> setuptools-58.1.0.dist-info
#> types_s3transfer-0.6.0.post4.dist-info 

# confirm these files are the same:
diff /tmp/s3transfer-test/lib/python3.9/site-packages/types_s3transfer-0.6.0.post4.dist-info/LICENSE /tmp/s3transfer-test/lib/python3.9/site-packages/LICENSE
# (no output = yes)
```

Possible solutions:

- package `LICENSE` into `s3transfer-stubs`
- not explicitly package `LICENSE`, and just rely on `types_s3transfer-0.6.0.post4.dist-info/LICENSE`

---

Thanks for `types-s3transfer`!

**Comment by vemel** [comment]
Thank you for the report!

I have just released `types-s3transfer 0.6.0.post5` with the issue fixed.

Please let me know if it works as it should.

**Comment by huonw** [comment]
Looks good, thanks for the quick fix!
